analytics/terraform/datahub-on-eks/install.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+read -p "Enter the region: " region
+export AWS_DEFAULT_REGION=$region
+
+echo "Initializing ..."
+terraform init || echo "\"terraform init\" failed"
+
+# List of Terraform modules to apply in sequence
+targets=(
+  "module.vpc"
+  "module.vpc_endpoints_sg"
+  "module.vpc_endpoints"
+  "module.eks"
+  "module.ebs_csi_driver_irsa"
+  "module.vpc_cni_irsa"
```

Comment:
Do you need to this resource as well?   "aws_eks_addon.vpc_cni"

Reply:
added dependency to the addon module to this resource to have the cni deployed before other addons.